mp/src/game/client/momentum/ui/spectate/mom_replayui.cpp

```diff
@@ -255,6 +253,8 @@ void C_MOMReplayUI::ShowPanel(bool state)
     if (pSpecUI && pSpecUI->IsVisible() && ipanel()->IsMouseInputEnabled(pSpecUI->GetVPanel()))
         SetMouseInputEnabled(true);
     if (state)
+        FIND_LOCALIZATION(m_pwReplayTime, "#MOM_ReplayTime");
+        FIND_LOCALIZATION(m_pwReplayTimeTick, "#MOM_ReplayTimeTick");
         MoveToFront();
```

Comment:
Wrap these in curlies:

```suggestion
    if (state)
    {
        FIND_LOCALIZATION(m_pwReplayTime, "#MOM_ReplayTime");
        FIND_LOCALIZATION(m_pwReplayTimeTick, "#MOM_ReplayTimeTick");
        MoveToFront();
    }
```